internal/sbi/processor/ampolicy.go

```diff
@@ -308,8 +300,8 @@ func (p *Processor) PostPoliciesProcedure(polAssoId string,
 }
 
 // Send AM Policy Update to AMF if policy has changed
-func (p *Processor) SendAMPolicyUpdateNotification(ue *pcf_context.UeContext,
-	PolId string, request models.PolicyUpdate,
+func (p *Processor) AMPolicyUpdateNotification(ue *pcf_context.UeContext,
```

Comment:
Hi @HanHongChen ,

The logic here should move to the consumer `SendAMPolicyUpdateNotification()`. 
We will not check the notifier status and result here.  (Should be in `Consumer`)

Reply:
fixed